cmd/operator/main.go

```diff
@@ -105,6 +104,25 @@ func main() {
 	}
 }
 
+func setupOperatorController(mgr manager.Manager) error {
+	apiClient, err := client.New(mgr.GetConfig(), client.Options{Scheme: mgr.GetScheme(), Mapper: mgr.GetRESTMapper()})
+	if err != nil {
+		setupLog.Error(err, "failed creating non cached client")
+		return err
```

Comment:
The setupLog is already done at the caller.

```suggestion
		return fmt.Errorf("failed creating non cached client: %w", err)
```

Reply:
:+1: updated